src/chimera/interfaces/weatherstation.py

```diff
@@ -21,100 +21,125 @@
-                  "wind_unit": Unit.KM_PER_H,
-                  "dew_point_unit": Unit.CELSIUS,
-                  "pressure_unit": Unit.PERCENTUAL,
-                  "rain_unit": Unit.MM_PER_H,
-
-                  "humidity_delta": 1,
-                  "temperature_delta": 1,
-                  "wind_delta": 1,
-                  "dew_point_delta": 1,
-                  "pressure_delta": 1,
-                  "rain_delta": 1,
-                  }
-
-    def humidity(self, deltaT=0, unit=Unit.PERCENTUAL):
-        pass
+    """
 
-    def temperature(self, deltaT=0, unit=Unit.CELSIUS):
-        pass
-
-    def wind_speed(self, deltaT=0, unit=Unit.M_PER_S):
-        pass
-
-    def wind_direction(self, deltaT=0, unit=Unit.DEG):
-        pass
+    __config__ = {"device": None,           # weather station device
+                  "model": "unknown",     # weather station model
+                  }
 
-    def dew_point(self, deltaT=0, unit=Unit.CELSIUS):
-        pass
 
-    def pressure(self, deltaT=0, unit=Unit.MM_HG):
-        pass
 
-    def rain(self, deltaT=0, unit=Unit.MM_PER_H):
+    """
+    Humidity units accepted by the interface.
+    """
+    __accepted_humidity_units__ = [
+        units.pct
+    ]
+
+    """
+    Temperature units accepted by the interface.
+    """
+    __accepted_temperature_units__ = [
+        units.Celsius,
+        units.Kelvin,
+        units.imperial.deg_F
+    ]
+
+    """
+    Speed units accepted by the interface.
+    """
+    __accepted_speed_units__ = [
+        units.meter / units.second,
+        units.kilometer / units.hour,
+        units.imperial.mile / units.hour,
+        units.imperial.foot / units.second,
+    ]
+
+    """
+    Direction units accepted by the interface.
+    """
+    __accepted_direction_unit__ = [
+        units.degree,
+        units.radian
+    ]
+
+    """
+    Pressure units accepted by the interface.
+    """
+    __accepted_pressures_unit__ = [
+        units.cds.mmHg,
+        units.bar,
+        units.cds.atm,
+        units.Pa]
+
+    def humidity(self, unit_out=units.pct):
+        """
+        Returns the 100% relative humidity (Default: Percentage).
+        :param unit: Unit in which the instrument should return the humidity.
+        :return: the humidity.
+        """
         pass
 
-    @event
-    def humidityChange(self, humidity, unit, delta):
+    def temperature(self, unit_out=units.Celsius):
+        """
+        Returns the temperature in the chosen unit (Default: Celsius).
+        :param unit:  Unit in which the instrument should return the temperature.
+        :return: the temperature.
+        """
         pass
 
-    @event
-    def temperatureChange(self, temperature, unit, delta):
+    def wind_speed(self, unit_out=units.kilometer/units.hour):
+        """
+        Returns the wind speed in the chosen unit (Default: Meters per second).
+        :param unit:  Unit in which the instrument should return the wind speed.
```

Comment:
You say in the doc that it is m/s, but on def is km/h. I think that the default should be m/s.